Offline/OfflinePlayer/objc/OfflinePlayer/Managers and Utilities/DownloadManager.m

```diff
@@ -241,8 +241,22 @@ - (void)downloadVideoFromQueue
      
      }];
     
+    // Prior to iOS 13 it was possible to download secondary tracks separately from the video itself.
+    // On iOS 13+ you must now download secondary tracks along with the video.
+    // The existing method for downloading videos is: `requestVideoDownload:parameters:completion:`
+    // You may still use this method on iOS 11 and 12.
+    // If you want to support iOS 13 and do not want to have any branching logic
+    // you can use the new method that is backwards compatible:
+    // `requestVideoDownload:mediaSelections:parameters:completion:`
+    
     __weak typeof(self) weakSelf = self;
+    AVURLAsset *avURLAsset = [self.offlineVideoManager urlAssetForVideo:video error:nil];
+    NSArray<AVMediaSelection *> *mediaSelections = @[avURLAsset.preferredMediaSelection];
+    if (@available(iOS 11.0, *)) {
+        mediaSelections = [avURLAsset allMediaSelections];
+    }
```

Comment:
Something I'd like to work in here is a demonstration of examining the `AVMediaSelections` returned by the `AVAsset`. Have a look at [this](https://bithub.brightcove.com/videocloud/agave/blob/develop/src/core/Source/BCOVOfflineVideoManager.m#L956-L972) and add something like it to display the media selection type (audible/legible) and `displayName`. Whatchyathink?